__tests__/hooks/useGroupInfo.test.js

```diff
@@ -691,10 +734,6 @@ describe('useGroupInfo should run correctly', () => {
     expect(result.current.newParticipant).toEqual([]);
   });
 
-  it('onReportGroup should run correctly', async () => {
-    const {result} = renderHook(() => useGroupInfo(), {wrapper});
-    await result.current.onReportGroup();
-  });
   it('handlePressContact type group hould run correctly', async () => {
     const {result} = renderHook(() => useGroupInfo(), {wrapper});
     await result.current.handlePressContact({
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The test for the `onReportGroup` function has been removed. If this function is still part of your codebase, you should ensure that it's covered by your tests.
```diff
-  it('onReportGroup should run correctly', async () => {
-    const {result} = renderHook(() => useGroupInfo(), {wrapper});
-    await result.current.onReportGroup();
-  });
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->